RELEASE_NOTES.md

```diff
@@ -1,5 +1,9 @@
 # Release notes
 
+## New in git-machete 3.12.3
+
+- fixed: release instructions in CONTRIBUTING.md are now correct
```

Comment:
```suggestion
- fixed: release to Homebrew doesn't crash the deployment process
```

Reply:
(both are arguably un-interesting to users, but still, let's put the more important one as a filler-up ;p)

Reply:
Hmm, i'll add both, @PawelLipski ok?

Reply:
'kay!

Reply:
I would say that `fixed: release instructions in CONTRIBUTING.md are now correct` is more important for the user as he doesn't care about deployment